resources/views/ama_list.blade.php

```diff
@@ -0,0 +1,100 @@
+@extends('layouts/admin')
+
+@section('title')
+	AMAs list
+@endsection
+
+@section('content')
+
+			<div class="col-sm-12">
+			 <form action="">
+
+			 	<div class="row">
+    				<div class="col">
+    				  <label>Date From:</label>
+    				</div>
+   					 <div class="col">
+     					<label>Date To:</label>
+    				</div>
+    				<div class="col">
+     					<label>Tags:</label>
+    				</div>
+  				</div>
+
+ 				 <div class="row">
+    				<div class="col">
+    				  <input input type="date"  name="date_to" class="form-control">
+    				</div>
+   					 <div class="col">
+     					 <input input type="date"  name="date_to" class="form-control">
+    				</div>
+    				<div class="col">
+     					 <select id="select_tags" name="tags" multiple="multiple" style="width: 100%">
+     					 	@foreach($tags as $tags)
```

Comment:
you shouldn't use the same variable name for the array and for the value in the iteration, it will mess up the logic. Use  `@foreach($tags as $tag)` instead.